src/telliot/contract/contract.py

```diff
@@ -171,9 +168,11 @@ async def write_with_retry(
                 print("write status: ", status)
 
                 # Exit loop if transaction successful
-                if tx_receipt and status.ok:
-                    transaction_receipts.append(tx_receipt)
-                    return transaction_receipts, status
+                if tx_receipt and status.ok and tx_receipt["status"] == 1:
+                    print(
+                        f"tx was successful! check it out at {self.node.explorer}/tx/{tx_receipt['transactionHash']}"  # noqa: E501
+                    )  # noqa: E501
```

Comment:
What's `noqa`?

Reply:
it's for ignoring the "E105: line too long" flake8 failure